### PR TITLE
Django Lab 1

### PR DESCRIPTION
**Pull request by ckee7**

### CODE REVIEW

File: code/chris/Django/Lab_1/grocery/grocery/views.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+def complete(request, pk):
+    selected_item = get_object_or_404(GroceryItem, pk=pk)
+    print(selected_item)
+    selected_item.task_completed = True
+    selected_item.completed_date = timezone.now()
+    selected_item.save()
+    # return render(request, 'grocery:index', completed_item=completed_item)
+    return HttpResponseRedirect(reverse('grocery:index'))
+
```
Comment (brucestull) [inline code/chris/Django/Lab_1/grocery/grocery/views.py:33]: * Need a view to 'incomplete' a grocery item:
    * `The user should be able to mark an item complete/incomplete and be able to delete an item.`

**Review by brucestull — CHANGES_REQUESTED** [review]
[Lab 1: Grocery List](https://github.com/PdxCodeGuild/class_swordfish/blob/main/3%20Django/labs/01%20Grocery%20List.md)
**Needs Work**
* Comments:
    * Application needs to have feature to 'incomplete' a grocery item.
        * `The user should be able to mark an item complete/incomplete and be able to delete an item.`

[Lab 2: URL Shortener](https://github.com/PdxCodeGuild/class_swordfish/blob/main/3%20Django/labs/02%20URL%20Shortener.md)
**Completed**
* Comments:
    * Lab does exaclty what's required.
    * Great, and simple, code.
    * Good job.

File: code/chris/Django/Lab_1/grocery/grocery/views.py (hunk 1 above)
Comment (ckee7) [reply code/chris/Django/Lab_1/grocery/grocery/views.py:37]: Done

**Review by brucestull — APPROVED** [review]
[Lab 1: Grocery List](https://github.com/PdxCodeGuild/class_swordfish/blob/main/3%20Django/labs/01%20Grocery%20List.md)
**Completed**

[Lab 3: Chirp (Twitter Clone)](https://github.com/PdxCodeGuild/class_swordfish/blob/main/3%20Django/labs/03%20Chirp.md)
**Completed**
* Comments:
    * Nice and simple css.
    * Code works great.
    * Understandable UI.

**Pull request merged**